### PR TITLE
fix: enable concurrent reconciles

### PR DESCRIPTION
**Pull request by evadnoob**
Enables concurrent reconciles, also requires changes in services

https://github.com/getoutreach/services/pull/382

### CODE REVIEW

File: templates/internal/appName/k8s/activity.go.tpl [hunk 1]
```diff
@@ -66,8 +67,9 @@ func (s *KubernetesService) Run(ctx context.Context) error { //nolint: funlen,ll
 	ctrl.SetLogger(logadapters.NewLogrLogger(ctx))
 
 	s.registerSchemes()
-
-	options := ctrl.Options{
+  
+    // manager options
+    mgrOptions := ctrl.Options{
```
Comment (AndrewWinterman) [inline templates/internal/appName/k8s/activity.go.tpl:72]: can we use the whole word? `manager` was not the first thing i thought of

**Review by AndrewWinterman — APPROVED** [review]
appears to be updating stencil based on updated apis in Services

File: templates/internal/appName/k8s/activity.go.tpl (hunk 1 above)
Comment (evadnoob) [reply templates/internal/appName/k8s/activity.go.tpl]: Ok, changed

**Review by AndrewWinterman — APPROVED** [review]
still looks fine

**Review by jaredallard — APPROVED** [review]
Seems solid. Will review the larger `services` part soon

**Pull request merged**